kernel/src/main/scala/org/apache/toree/boot/layer/ComponentInitialization.scala

```diff
@@ -94,6 +98,18 @@ trait StandardComponentInitialization extends ComponentInitialization {
 
   }
 
+  def initializeSparkContext(config:Config, kernel:Kernel) = {
+    // TOREE:425 Spark cluster mode requires a context to be initialized before
+    // it register the application as Running
+    if ( SparkUtils.isSparkClusterMode(kernel.sparkConf) ) {
```

Comment:
Nit: Are the spaces needed? I think this is non-standard for Scala or Java projects.